packages/fetchai/agents/aw5/aea-config.yaml

```diff
@@ -0,0 +1,160 @@
+agent_name: aw5
+author: fetchai
+version: 0.1.0
+license: Apache-2.0
+description: An agent that participates in Agent World 5.
+aea_version: '>=1.0.0, <2.0.0'
+fingerprint: {}
+fingerprint_ignore_patterns: []
+connections:
+- fetchai/http_client:0.22.0
+- fetchai/ledger:0.18.0
+- fetchai/p2p_libp2p:0.21.0
+- fetchai/soef:0.22.0
+contracts: []
+protocols:
+- fetchai/contract_api:1.0.0
+- fetchai/default:1.0.0
+- fetchai/fipa:1.0.0
+- fetchai/http:1.0.0
+- fetchai/ledger_api:1.0.0
+- fetchai/oef_search:1.0.0
+- fetchai/register:1.0.0
+- fetchai/signing:1.0.0
+- fetchai/state_update:1.0.0
+skills:
+- fetchai/generic_seller:0.25.0
+- fetchai/registration_aw1:0.10.0
+- fetchai/simple_data_request:0.11.0
+- fetchai/simple_seller:0.11.0
+- fetchai/simple_service_registration:0.20.0
+- fetchai/simple_service_search:0.9.0
+default_connection: fetchai/p2p_libp2p:0.21.0
+default_ledger: fetchai
+required_ledgers:
+- fetchai
+default_routing:
+  fetchai/http:1.0.0: fetchai/http_client:0.22.0
+  fetchai/ledger_api:1.0.0: fetchai/ledger:0.18.0
+  fetchai/oef_search:1.0.0: fetchai/soef:0.22.0
+connection_private_key_paths: {}
+private_key_paths: {}
+logging_config:
+  disable_existing_loggers: false
+  version: 1
+  formatters:
+    standard:
+      format: '[%(levelname)s]: %(message)s'
+    extra:
+      format: '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+  handlers:
+    logfile:
+      class: logging.FileHandler
+      formatter: extra
+      level: DEBUG
+      filename: all.log
+    console:
+      class: logging.StreamHandler
+      formatter: standard
+      level: INFO
+  loggers:
+    aea:
+      handlers:
+      - logfile
+      - console
+      level: DEBUG
+      propagate: false
+dependencies:
+  aea-ledger-ethereum:
+    version: <2.0.0,>=1.0.0
+  aea-ledger-fetchai:
+    version: <2.0.0,>=1.0.0
+---
+public_id: fetchai/registration_aw1:0.10.0
+type: skill
+models:
+  strategy:
+    args:
+      announce_termination_key: aw-registration
+      ethereum_address: PUT_YOUR_ETHEREUM_ADDRESS_HERE
+      developer_handle: PUT_YOUR_DEVELOPER_HANDLE_HERE
+      signature_of_fetchai_address: PUT_YOUR_SIGNATURE_HERE
+      tweet: PUT_THE_LINK_TO_YOUR_TWEET_HERE
+      whitelist:
+      - PUT_WHITELIST_ADDRESSES_HERE
+---
+public_id: fetchai/simple_service_registration:0.20.0
+type: skill
+models:
+  strategy:
+    args:
+      location:
+        latitude: 51.5194
+        longitude: 0.127
```

Comment:
Hm this will not work..let's discuss

Reply:
Easiest option, first registration skill runs with location of registration agent, then when that's done we activate seller skill and register agent in new location. Alternatively, we have no registration and buyer buys from all agents (as we now have main net we no longer need eth address)